mkt/users/models.py

```diff
@@ -63,24 +62,17 @@ def widget_attrs(self, widget):
                 'data-src': lazy_reverse('users.ajax')}
 
 
-AbstractBaseUser._meta.get_field('password').max_length = 255
```

Comment:
We have no data anymore in this field, thanks to https://github.com/mozilla/zamboni/blob/master/migrations/839-cleanup-users.sql

You're right though.